### PR TITLE
bump prispa version

### PR DESCRIPTION
**Pull request by kanolato**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x ] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x ] Chart Version bumped
- [ x] Variables are documented in the README.md

### CODE REVIEW

**Comment by cpanato** [comment]
@kanolato bump the chart version and check if need to update the readme as well and sign the DCO

File: stable/prisma/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: prisma
-version: 1.1.0
-appVersion: 1.20
+version: 1.1.1
+appVersion: 1.29
```
Comment (cpanato) [inline stable/prisma/Chart.yaml]:
```suggestion
appVersion: 1.29.1
```

Comment (cpanato) [reply stable/prisma/Chart.yaml]: also need update README.md

**Comment by cpanato** [comment]
@kanolato still missing to sign the DCO

**Comment by kanolato** [comment]
@cpanato DCO signed :)

**Pull request closed without merging**